package.json

```diff
@@ -4,19 +4,20 @@
   "type": "module",
   "scripts": {
     "dev": "concurrently \"yarn dev:css\" \"npx @11ty/eleventy --serve --formats=md,html,png,jpg,jpeg,ico,webp,gif\"",
-    "dev:css": "tailwindcss -i index.css -o _site/assets/css/index.css --watch --postcss",
-    "build:css": "tailwindcss -i index.css -o _site/assets/css/index.css --postcss",
+    "dev:css": "npx @tailwindcss/cli -i index.css -o _site/assets/css/index.css --watch --postcss",
```

Comment:
The CLI part of Tailwind is now a separate package. You shouldn't need this if you have Vite or Next in the build chain for the app.